### DISCUSSION TITLE
Hi folks What s the easiest way to clone [Jun 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2487] [Discord Threads]
> 🙋‍♂️ **Remco**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi! Could you be more precise?  
#  
> 🙋‍♂️ **FrancyMak**  
#### Do you want only the schema or all-data?  
#  
> 🙋‍♂️ **Remco**  
#### both indeed  
#  
> 👨‍🦲 **dstala**  
#### <@110825093977870336>, currently its not possible to clone database/ project directly from UI. We will have to take a backup from the backend by using SQL commands (SQL dump for example)  
#  
> 🙋‍♂️ **Remco**  
#### ah, thnx. good to know. thnx .  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987317298949922896)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
